src/styles/modules/_modules.scss

```diff
@@ -7,3 +7,4 @@
 @import 'divs';
 @import 'icons';
 @import 'dropdown';
+@import "button-status";
```

Comment:
We are importing it here again, this is the right place though, let's remove it from base.scss